tests/backend/test_jdbc.py

```diff
@@ -37,6 +37,14 @@ def test_close(test_mp, caplog):
         'Database connection could not be closed or was already closed'
 
 
+def test_connect_message(caplog, request):
+    sample_props = os.path.join(request.fspath.dirname,
+                                '../data/testdb/test.properties.sample')
```

Comment:
Please use the test_data_path fixture: https://github.com/iiasa/ixmp/blob/077387fbabb2536927b723b4e2004810259a3af5/tests/conftest.py#L42-L45
as elsewhere in the test suite. Something like:
```
def test_connect_message(caplog, test_data_path):
    sample_props = test_data_path / 'testdb' / 'test.properties.sample'
    # etc.
```

Reply:
thanks for suggestion - always struggling to reuse fixtures